packages/python/plotly/tox.ini

```diff
@@ -81,25 +81,25 @@ deps=
 basepython={env:PLOTLY_TOX_PYTHON_27:}
 commands=
     python --version
-    pytest {posargs} -x plotly/tests/test_core
+    pytest {posargs} plotly/tests/test_core
 
 [testenv:py35-core]
 basepython={env:PLOTLY_TOX_PYTHON_35:}
 commands=
     python --version
-    pytest {posargs} -x plotly/tests/test_core
+    pytest {posargs} plotly/tests/test_core
 
 [testenv:py36-core]
 basepython={env:PLOTLY_TOX_PYTHON_36:}
 commands=
     python --version
-    pytest {posargs} -x plotly/tests/test_core
```

Comment:
why did you remove the `-x` ? Exiting on first failing test is good for the environment :-).

Reply:
I needed to be able to see if I fixed a test even if an "earlier" one failed. Looking at the timings, it seems to me that the container setup etc takes more time than the test suites proper, so while we have the container spun up we may as well extract as much information as possible about the run, is my feeling :)